projects/js-packages/image-guide/composer.json

```
@@ -0,0 +1,52 @@
+{
+	"name": "automattic/jetpack-image-guide",
+	"description": "Go through the dom to analyze image size on screen vs actual file size.",
+	"type": "library",
+	"license": "GPL-2.0-or-later",
+	"require": {},
+	"require-dev": {
+		"yoast/phpunit-polyfills": "1.0.4",
+		"automattic/jetpack-changelogger": "@dev"
+	},
+	"autoload": {
+		"classmap": [
+			"src/"
+		]
+	},
+	"scripts": {
+		"build-development": [
+			"pnpm run build"
+		],
+		"build-production": [
+			"NODE_ENV=production pnpm run build"
+		],
+		"watch": [
+			"Composer\\Config::disableProcessTimeout",
+			"pnpm run watch"
+		],
+		"test-js": [
+			"pnpm run test"
+		],
+		"test-coverage": [
+			"pnpm run test-coverage"
+		]
+	},
+	"repositories": [
+		{
+			"type": "path",
+			"url": "../../packages/*",
+			"options": {
+				"monorepo": true
+			}
+		}
+	],
+	"minimum-stability": "dev",
+	"prefer-stable": true,
+	"extra": {
+		"mirror-repo": "Automattic/jetpack-image-guide",
```

If we have a mirror repo, we may as well set up auto-publish to NPM:

```suggestion
		"npmjs-autopublish": true,
		"mirror-repo": "Automattic/jetpack-image-guide",
```

---

Ok, I'll add that in a subsequent PR that @eAdnan007 can approve :)